Consortium.md

```diff
@@ -0,0 +1,51 @@
+# CreateJS Consortium
+
+The GSkinner team is handing the CreateJS project over to the community.
```

Comment:
Perhaps: The GSkinner team is encouraging the community to get involved in in the maintenance and direction of CreateJS.

I am not sure they are ready to give it up completely.  Probably a side-by-side approach may be the way to look at it.  So the title - "Start a consortium to take over CreateJS" may be a bit dramatic - although it does sound like what consortia do!  How about "Inititial Proposal to help maintain CreateJS".

From a meta standpoint... can we edit this document directly anywhere?  Or do we just provide comments?  Also... what is the "Start a review" button and when would we use it?

Reply:
Good point. I had made some assumptions about what was happening. Happy to reword. 

To edit, you can certainly submit a PR to merge into my PR branch on my jedateach github. I might somehow be able to give you permission to edit my branch directly from this ui.